quantum/backlight/backlight_arm.c

```diff
@@ -0,0 +1,214 @@
+#include "quantum.h"
+#include "backlight.h"
+#include <hal.h>
+#include "debug.h"
+
+// TODO: remove short term bodge when refactoring BACKLIGHT_CUSTOM_DRIVER out
+#ifdef BACKLIGHT_PIN
+
+#    if defined(STM32F1XX) || defined(STM32F1xx)
+#        define USE_GPIOV1
+#    endif
+
+// GPIOV2 && GPIOV3
+#    ifndef BACKLIGHT_PAL_MODE
+#        define BACKLIGHT_PAL_MODE 2
+#    endif
+
+// GENERIC
+#    ifndef BACKLIGHT_PWM_DRIVER
+#        define BACKLIGHT_PWM_DRIVER PWMD3
+#    endif
+#    ifndef BACKLIGHT_PWM_CHANNEL
+#        define BACKLIGHT_PWM_CHANNEL 3
```

Comment:
Should this be zero or 1 indexed?

STM docs are 1 based
ChibiOS is zero

At the moment i have aligned to docs but happy to go either way